FirebaseDatabase/Sources/Core/FSyncTree.m

```diff
@@ -448,6 +457,42 @@ - (NSArray *)applyTaggedServerRangeMergeAtPath:(FPath *)path
     }
 }
 
+- (NSNumber *)registerQuery:(FQuerySpec *)query {
```

Comment:
Query is used a lot, including for listeners. Should we call this something like "registerGet"?